### ISSUE TITLE
Zen crashes when trying to access chrome://browser/content/browser.xhtml

### ISSUE THREAD

**Issue by MoltenMonster** [zen-browser/desktop#3941]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

In Firefox you can go to chrome://browser/content/browser.xhtml to see the whole browser UI in a webpage and use dev tools to inspect it and use it to work on your userChrome.css file. when opening the URL in Zen, it immediately crashes.

![CleanShot 2024-12-22 at 11 28 57@2x](https://github.com/user-attachments/assets/94bc270e-0aa1-4ac6-9087-9d8155802ad9)

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.3 (Firefox 133.0.3)

### What platform are you seeing the problem on?

macOS - aarch64

**Comment by Numline1** [comment]
Can confirm, this crashes Zen even with devtools.debugger.remote-enabled set to true, however @Oneechan69 , you can use `Ctrl (or CMD) + Alt + Shift + I` to open devtools and inspect any browser element, granted you have `devtools.debugger.remote-enabled` set to true in `about:config`.

**Comment by MoltenMonster** [comment]
Yeah, though using the URL would be more convenient as I wouldn't have to switch windows and have the entire thing in one browser tab.

**Comment by mr-cheffy** [comment]
You are not supposed to access it...

**Comment by MoltenMonster** [comment]
@mauro-balades Then this is one thing that Zen Browser is missing from Firefox, this would be really nice for working on CSS, especially since there's a store for Zen Mods.

**Comment by JSGRD22** [comment]
![image](https://github.com/user-attachments/assets/4f6f57a0-6c19-4554-ab0c-45295d357043)
cannot reproduce (on windows)

**Comment by MoltenMonster** [comment]
Still have this issue on `1.7t (Firefox 134.0.1) (2025-01-15) (aarch64)` macOS.

**Comment by heyitszenithyt** [comment]
> @mauro-balades Then this is one thing that Zen Browser is missing from Firefox, this would be really nice for working on CSS, especially since there's a store for Zen Mods.

Just activate the Browser Toolbox.

https://docs.zen-browser.app/guides/live-editing

**Comment by MoltenMonster** [comment]
I know but that operates on actual browser windows, and there's no `CTRL Z` / `CMD Z` to undo. This page would allow you to inspect the browser UI in its own tab.

**Comment by D-Maxwell** [comment]
having enabled remote debugging, the hotkey `ctrl alt shift i` refuses to open the browser development tools.

should i open a new issue for this ?

**Comment by MoltenMonster** [comment]
Yeah, the browser toolbox is a different thing from this page. The toolbox window works fine for me.

**Comment by D-Maxwell** [comment]
> Yeah, the browser toolbox is a different thing from this page. The toolbox window works fine for me.

thank you, and sorry. i only realised this referred to a different thing past having commented the above ; besides, i have found what the issue was. it would seem --as reported in the browser console-- that, depending on the locale, the shortcut meant to invoke said browser toolbox is unable to be assigned to this action, possibly being overwritten ? it doesn't seem to do anything, strangely.
however, i have noticed the existence of a graphical method to palliate this issue, through the browser toolbox button under the more tools section.

i may add that accessing the page actually relevant to this issue does not cause my browser to crash. (windows, 1.7.2b)
seems like a macOS specific issue.

**Comment by MoltenMonster** [comment]
I still have this issue

**Comment by mr-cheffy** [comment]
## Closing this issue as part of a repo cleanup

Due to technical-debt we had in the past, we are currently clearing out old, duplicate, stale, and non-actionable issues to improve issue tracking and make it easier to focus on fixing real bugs.

If this issue is still relevant, please open a new issue using our updated issue template, which will help us categorize and address it more efficiently.

Thanks for your understanding and for helping us improve the project! 🚀

This issue can be reopened at: https://github.com/zen-browser/desktop/issues/new?template=bug_report.yml&title=Zen%20crashes%20when%20trying%20to%20access%20chrome%3A%2F%2Fbrowser%2Fcontent%2Fbrowser.xhtml%0A
